frontend/fingertips-frontend/playwright/testHelpers.ts

```diff
@@ -87,7 +87,7 @@ export function getScenarioConfig(
       componentLocator: ChartPage.lineChartComponent,
       componentProps: {
         hasConfidenceIntervals: true,
-        hasBenchmark: true,
+        // hasBenchmark: true,
```

Comment:
Temporary disable the benchmark e2e testing after moving the benchmark area select to top of the view page.